app/controllers/admin/site_settings_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Admin::SiteSettingsController < Admin::ApplicationController
```

Comment:
__[rubocop]__
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->